src/action/log.js

```diff
@@ -15,6 +15,9 @@ class LogAction {
     _ipcRenderer = ipcRenderer;
     _ipcRenderer.on('logs', (event, arg) => {
       store.logs.push(arg);
+      if (store.logs.length > 100) {
+        store.logs.splice(0, store.logs.length - 100);
+      }
```

Comment:
This will return the first 100 not the last:
```
> ['foo', 'bar', 'baz', '42'].splice(0, 4 - 2)
returns [ 'foo', 'bar' ]
```

If we only had unit tests 😉

Reply:
Lol, point taken xD However the code is correct -- the return value of `splice` is the removed list items

Reply:
> the return value of splice is the items that were removed from the list

Ah ok. My bad. Well unit tests wouldn't hurt either way.